File: python/7 kyu/407.py
# https://www.codewars.com/kata/5cd5ba1ce4471a00256930c0/train/python

# Passed

def solution(n, d):
    result = []
    if d > 0:
        digits = [int(digit) for digit in str(n)]
        result = digits[-d:]
    
    return result

output = solution(123767, 4)
print(output)

File: nodejs/7 kyu/407.js
// https://www.codewars.com/kata/55736129f78b30311300010f/train/javascript

// Passed

function pattern(n) {
  let rowsReversed = [];
  let row = "";
  for (let num = n; num >= 1; num--) {
    row = String(num) + row;
    rowsReversed.push(row);
  }

  let rows = rowsReversed.reverse();
  let result = rows.join("\n");
  return result;
}

const output = pattern(5);
console.log(output);